### PR TITLE
Add new action for TM1651

### PR DESCRIPTION
**Pull request by freekode**
## Description:
Added new action for TM1651. Level can be set as percent and as simple value from 0 to 7

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#920

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

**Comment by freekode** [comment]
Waiting for review

File: components/tm1651.rst [hunk 1]
```diff
@@ -72,6 +92,40 @@ Configuration variables:
 - **brightness** (**Required**, :ref:`templatable <config-templatable>`, int): There is three levels of brightness
   (``1``, ``2`` or ``3``) from lowest to highest brightness.
 
+.. _tm1651-turn_off:
+
+``tm1651.turn_on`` Action
```
Comment (glmnet) [inline components/tm1651.rst]:
```suggestion
``tm1651.turn_off`` Action
```

File: components/tm1651.rst [hunk 2]
```diff
@@ -72,6 +92,40 @@ Configuration variables:
 - **brightness** (**Required**, :ref:`templatable <config-templatable>`, int): There is three levels of brightness
   (``1``, ``2`` or ``3``) from lowest to highest brightness.
 
+.. _tm1651-turn_off:
+
+``tm1651.turn_off`` Action
+-------------------------
```
Comment (glmnet) [inline components/tm1651.rst]:
```suggestion
--------------------------
```

**Comment by glmnet** [comment]
Great

**Pull request merged**